index.js

```diff
@@ -9,10 +9,61 @@ function ask(questionText) {
 
 start();
 
+let min = 1;
+let max = 100;
+let guess;
+let secretNumber;
+
 async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
```

Comment:
You don't need to prompt the user for a secret number here. The secret number will be stored in the user's head.